avm/utl/types/avm-common-types/main.bicep

```diff
@@ -386,11 +386,11 @@ type customerManagedKeyWithAutoRotateType = {
   @description('Required. The name of the customer managed key to use for encryption.')
   keyName: string
 
-  @description('Optional. The version of the customer managed key to reference for encryption. If not provided, using version as per \'autoRotationDisabled\' setting.')
+  @description('Optional. The version of the customer managed key to reference for encryption. If not provided, using version as per \'autoRotationEnabled\' setting.')
```

Comment:
Sounds good. Btw (for tomorrow) the reason I pivoted towards 'disabled' & default `false` was because I felt it's the more natural way of using a 'switch' parameter. :)

Reply:
Interesting, I'm naturally inclined towards the opposite 😆 
Disabled & false sounds like a double negation to me and not intuitive.
I'd like to use positive statements for feature switches and propose the team to convert this AVM terraform spec to a shared spec https://azure.github.io/Azure-Verified-Modules/specs/terraform/#id-tfnfr16---category-code-style---variable-naming-rules

Reply:
Out of curiosity I just checked the repo (regex `\s{2}[a-zA-Z]+: bool`) and lo and behold - it's not consistently done at all 😄 

Anyways, my thinking went something like this
- If a property/parameter's default is `true`, then I'd expect a switch to disable it
- If a property/parameter's defautl is `false`, then I'd expect a switch to enable it

But maybe that makes literally no sense 😄

Reply:
It would, correct. And I guess that's already true - which is why it is so important for boolean parameters to specify it if they have a default - telling the user what they'd be enabling / disabling. 
Let's see if we can find a new timeslot.

Reply:
Let's update it as per your suggestion. 
I'll go ahead an update my PR for the spec accordingly: https://github.com/Azure/Azure-Verified-Modules/pull/1683